src/Omit/README.md

```diff
@@ -0,0 +1,21 @@
+# Omit
+
+A helper to omit by object's props.
+
+## Usage
+
+```js
+import type {Omit} from 'flown'
+
+type Original = {a: 1, b: 2, c: 3, d: 4};
+type Omitted = Omit<Original, ['a', 'b']>; 
+
+// type of Omitted: {c: 3, d: 4}
```

Comment:
if fact, the Type will be `{c: 3, d: 4, a: void, b: void}`, because I think it's impossible to remove properties from the type at this time

but it's possible to do by hands, like `$Diff<Omitted, {a: void, b: void}>`

Reply:
I agree, done it.